src/test/functional/features/opening-hours.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Opening Hours
+
+  Background:
+    Given I am on new browser
+    Given I am on FACT homepage
+    And I am on the admin portal sign in page
+    When I fill in the Username and Password fields with my authenticated credentials "hmcts.super.fact@gmail.com" "Pa55word11"
+    And click the Sign In button
+    Then I can view the courts or tribunals in a list format
+    And they are in alphabetical order
+    When I click edit next to a chosen court or tribunal
+    Then I am redirected to the Edit Court page for the chosen court
+    When I click the opening hours tab
+    Then I can view the existing opening hours
+
+  Scenario: Add new opening hours
+    When I enter new opening hours entry
+    And I click the Add button
+    And I click save
+    Then a green update message is displayed
+
+  Scenario: Prevent empty entries being added
+    When I enter a blank opening hours entry
+    And I click the Add button
+    And I click save
+    Then an error message is displayed
+
+  Scenario: Remove opening hours
+    When I click the remove button under an opening hours entry
+    Then I click save
+    Then a green update message is displayed
```

Comment:
Might be worth splitting the following into two for improved clarity: 

When('I click the remove button under an opening hours entry', async () => {
  const numOpeningTimes = await I.countElement('#openingTimesTab fieldset');

  const selector = 'button[name="deleteOpeningHours"]';
  const elementExist = await I.checkElement(selector);
  expect(elementExist).equal(true);
  await I.click(selector);

  const updatedNumOpeningTimes = await I.countElement('#openingTimesTab fieldset');
  expect(numOpeningTimes - updatedNumOpeningTimes).equal(1);
});

Reply:
How do we get the number of expected items for use in the new 'Then' clause?